src/components/pages/doc.js

```diff
@@ -1,48 +1,46 @@
-import React from 'react';
-import autoBind from 'auto-bind';
 import PropTypes from 'prop-types';
```

Comment:
I wish you wouldn't have done that since I'm completely reworking this page and it's working in a different way. Let's see (I'm all for going functional, it's just bad timing). :)))

Reply:
I've already asked if I can do this last week, then I had to also rework this PR due to conflicts 😬